### ISSUE TITLE
bug: Invalid Checksum on Download

### ISSUE THREAD

**Issue by ScottFinlaysonGMSL** [aquasecurity/tfsec#1887]
**Describe the bug**

We download a specific version of tfsec within our pipeline and confirm the binary against the checksum within the release notes.

The version/tag we download is `v1.8.0` and the checksum we were expecting was `6f0ffcc204a73991fb2b47561bbdf473e98a9566036c0f36b95d9574fd5ab4ce`.

At Sep 6, 2022 11:51 AM (UTC), we confirmed that the downloaded binary received had a checksum of `3590bcb90a75c32ba8b10d692d26838caedbc267a57db23931694abc9598c873` causing our pipeline to stop to avoid using anything that may have been compromised.

The command to download the tfsec binary:
```
curl -sL --output tfsec "https://github.com/aquasecurity/tfsec/releases/download/v1.8.0/tfsec-linux-amd64"
```

**To Reproduce**
I've retried the failing pipelines today and they are now passing but with no explanation as to what happened to cause the checksum to be incorrect. It's unclear if this would happen again but it is the first time we've seen this happen.

**Expected behavior**

Checksums should have matched. If there was an accident resulting in a temporary mismatch of the checksums then I'd expect a blog post or something similar which explained what happened but I couldn't find anything.

**Output of your tfsec command with --debug flag**

Unable. No longer have the "bad" binary that was downloaded.

**System Info**

 - tfsec version: `v1.8.0`
 - terraform version: `1.1.5`
 - OS: `Ubuntu 20.04`

**Example Code**

The command to download the tfsec binary:
```
curl -sL --output tfsec "https://github.com/aquasecurity/tfsec/releases/download/v1.8.0/tfsec-linux-amd64"
```

Checksum calculated via
```
sha256sum tfsec | awk '{ print $1 }'
```

Checksum compared to is stored locally so as to avoid any potential manipulation with the original release.

**Additional context**

**Comment by owenrumney** [comment]
Hi @ScottFinlaysonGMSL - I've just tested that binary and it has the expected as you say. We don't replace binaries of existing tags and none of our build process would allow this.

Aside from that, 1.8.0 is from March so we wouldn't manually attempt to replace this now anyway.

Could your pipeline have been changed to erroneously download a different version?

**Comment by ScottFinlaysonGMSL** [comment]
Yeah that was my first thought and I've looked at the script used (we use AWS Codebuild so we keep a history of the config used at that time) and it's _exactly_ the same as it was for the instance that passed today.
